packages/p/playerctl/package.yml

```diff
@@ -18,3 +19,4 @@ build      : |
 install    : |
     %ninja_install
     rm -f $installdir/%libdir%/*.a
+
```

Comment:
Is this an extra newline?

Reply:
Yes, lite-xl inserts a new line when I save. Should I remove it?

Reply:
The file already ends with a newline, from what I can see. No need to have two newlines.

Reply:
On my end it looks like it never had one. It just ended at the rm command

Reply:
I double-checked in Zed, and yeah, there was already a newline at the end, so this PR adds an extra unneeded line.

Reply:
For some reason github was hiding that line. Sorry about that.